src/lib/collectors/utils/requester.ts

```diff
@@ -31,7 +31,8 @@ const defaults = {
         'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/56.0.2924.87 Safari/537.36'
     },
     jar: true,
-    time: true
+    time: true,
+    timeout: 3000
```

Comment:
This value is way too small. Any specific reason why you are changing this value instead of relying in what the OS does?